widget/embedded/src/hooks/useSyncUrlAndStore/useSyncUrlAndStore.ts

```diff
@@ -124,18 +124,16 @@ export function useSyncUrlAndStore() {
       const fromBlockchain = blockchains.find(
         (blockchain) => blockchain.name === searchParams.fromBlockchain
       );
-      const fromToken = searchParamToToken(
-        tokens,
-        searchParams.fromToken,
-        fromBlockchain ?? null
+
+      const fromToken = findToken(
+        getTokenAsset(searchParams.fromToken, fromBlockchain ?? null)
       );
       const toBlockchain = blockchains.find(
         (blockchain) => blockchain.name === searchParams.toBlockchain
       );
-      const toToken = searchParamToToken(
-        tokens,
-        searchParams.toToken,
-        toBlockchain ?? null
+
+      const toToken = findToken(
+        getTokenAsset(searchParams.toToken, fromBlockchain ?? null)
```

Comment:
Please use `toBlockchain` instead of `fromBlockchain`

When the source and destination tokens are selected and placed in the URL, please refresh once and make sure everything is working correctly.

Reply:
fixed